utils/build.js

```diff
@@ -90,15 +90,19 @@ exports.mergeCrowdsaleContract = async function (tokenId) {
     const MERGE_DIR_PATH = path.join(PWD, '/../.build/', tokenId, '/', MERGE_DIR_NAME);
 
     const shell = require('shelljs');
-    shell.cd(ORACLES_COMBINE_DIR_PATH);
 
     let CROWDSALE_FILE_PATH = path.join(BUILD_DIR_PATH, '/', tokenId, '/', CONTRACT_DIR_NAME,'/', CROWDSALE_FILE_NAME);
     log.info(CROWDSALE_FILE_PATH);
-    if (shell.exec('npm start ' + CROWDSALE_FILE_PATH).code !== 0) {
-        shell.echo('Error: merge file failed');
-        shell.exit(1);
+    let npmRes = shell.exec('npm start --prefix '+ ORACLES_COMBINE_DIR_PATH + ' ' + CROWDSALE_FILE_PATH);
+    if (npmRes.code !== 0) {
+       log.error('oracle compbine npm install failed');
+       throw new Error("failed to merge contract");
     } else {
-        shell.mv(ORACLES_COMBINE_OUT_DIR_PATH + '/CrowdSale_flat.sol',  path.join(MERGE_DIR_PATH, '/', CROWDSALE_FILE_NAME));
+        let mvRes = shell.mv(ORACLES_COMBINE_OUT_DIR_PATH + '/CrowdSale_flat.sol',  path.join(MERGE_DIR_PATH, '/', CROWDSALE_FILE_NAME));
+        if (mvRes.code !==0) {
+            log.error('failed to move merged file');
+            throw new Error("failed to merge contract");
```

Comment:
failed to move merged file here?

Reply:
ya just in case some unexpected situation like the folder permission issue